app/interactors/concerns/fs_sanitizable.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "zaru"
```

Comment:
All gems listed in the Gemfile get required [here](https://github.com/t3-innovation-network/desm/blob/e7e797bbf717602a3f4b17cc8ae40dceeea52f18/config/application.rb#L19) unless their `require` flag is set to `false`, so this line is unnecessary.